### PR TITLE
initial add of getter/setter for Method of Death Date Determination

### PR DESCRIPTION
**Pull request by saulakravitz**
Please review annotations

### CODE REVIEW

File: VRDR/DeathRecord.cs [hunk 1]
```diff
@@ -5819,6 +5819,46 @@ public string DeathTime
...
+        /// <para>Dictionary&lt;string, string&gt; code = new Dictionary&lt;string, string&gt;();</para>
+        /// <para>code.Add("code", "exact");</para>
+        /// <para>code.Add("system", CodeSystems.DateOfDeathDeterminationMethods);</para>
+        /// <para>code.Add("display", "exact");</para>
+        /// <para>ExampleDeathRecord.DateOfDeathDeterminationMethod = code;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Date of Death Determination Method: {ExampleDeathRecord.DateOfDeathDeterminationMethod['display']}");</para>
+        /// </example>
+
+       [Property("DateOfDeathDeterminationMethod", Property.Types.Dictionary, "Death Investigation", "Date of Death Determination Method.", true, IGURL.DeathDate, true, 25)]
```
Comment (mitre-jim) [inline VRDR/DeathRecord.cs:5839]: Should this be `IGURL.DeathDate` or should it use the `CodeSystems` IG URL that was added?

Comment (saulakravitz) [reply VRDR/DeathRecord.cs:5839]: The annotation references the profile that this data element is part of....right @pkmitre ?

Comment (pkmitre) [reply VRDR/DeathRecord.cs:5839]: IGURL.DeathDate is correct

File: VRDR/DeathRecord.cs [hunk 2]
```diff
@@ -5819,6 +5819,46 @@ public string DeathTime
...
+       [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='81956-5').method", "")]
+        public Dictionary<string, string> DateOfDeathDeterminationMethod
+        {
+            get
+            {
+                if (DeathDateObs != null && (DeathDateObs.Method as CodeableConcept) != null)
+                {
+                    return CodeableConceptToDict(DeathDateObs.Method);
+                }
+                return EmptyAddrDict();
```
Comment (pkmitre) [inline VRDR/DeathRecord.cs:5849]: Why are you returning an EmptyAddrDict()? Don't you want EmptyCodeableDict()?

**Review by pkmitre — CHANGES_REQUESTED** [review]

File: VRDR/DeathRecord.cs (hunk 2 above)
Comment (saulakravitz) [reply VRDR/DeathRecord.cs]: Good catch....

**Review by pkmitre — APPROVED** [review]

**Pull request merged**